--- forester.1	2015-05-11 18:39:43.817829363 +0000
+++ forester.1.a	2015-05-11 18:39:23.459829343 +0000
@@ -3,6 +3,8 @@
 .SH NAME
 forester \- manage a forest of git trees
 .SH DESCRIPTION
+Clone multiple Git Repos from a single source file. Yet another "for" loop for git. Currently Git Forest must be managed in WMS.
+.B
 Common Commands
 .TP
 branch
@@ -33,6 +35,24 @@ push
 Push a local git forest to a remote repository
 .SH SEEALSO
 forester help <subcommand>
+.TP
+\fIforester\-init\fP(1)
+.TP
+\fIforester\-merge\fP(1)
+.TP
+\fIforester\-config\fP(1)
+.TP
+\fIforester\-pull\fP(1)
+.TP
+\fIforester\-checkout\fP(1)
+.TP
+\fIforester\-branch\fP(1)
+.TP
+\fIforester\-push\fP(1)
+.TP
+\fIforester\-clone\fP(1)
 .SH BUGS
 file issues or bugs https://opensource.sas.com/its
 .SH AUTHORS
